templates/repo/diff/new_review.tmpl

```diff
@@ -22,9 +22,21 @@
 					</div>
 				{{end}}
 				<div class="ui divider"></div>
-				<button type="submit" name="type" value="approve" {{if and $.IsSigned ($.Issue.IsPoster $.SignedUser.ID)}} disabled {{end}} class="ui submit green tiny button btn-submit">{{$.locale.Tr "repo.diff.review.approve"}}</button>
+				{{if and $.IsSigned ($.Issue.IsPoster $.SignedUser.ID)}}
```

Comment:
We can create a in-template variable for this value, avoiding re-computation.

```handlebars
{{$showSelfTooltip := (and $.IsSigned ($.Issue.IsPoster $.SignedUser.ID))}}
```

And then use as `{{if $showSelfTooltip}}`

Reply:
Done